DirectEmployers/MyJobs
# [PD-4012]  new analytics dimensions and UI

## What does this PR do?
Offers a new way to look at analytics data. 

## How should this be manually tested?
Go to New Analytics and play with the dimension groups, different column combos and make sure they all update the table correctly (selecting a new intention from the dropdown should change data columns, checking a new column on the left should add it to the table etc); make sure export still works. Note: timeline and bar charts are temporarily disabled in this PR. Talk to @aljordan82 if you have any concerns or questions about those.

## Any background context you want to provide?
New "intentions" are added to a special report intention called `analytics-pa` (pa = pick any).
The new intentions are based on data columns. When you look at the code, these intentions are actually defined as categories on the back end, I have a plan to refactor once we are ready to get rid of the old intentions. Doing this for now lowers the risk of breaking current reports.
Each new intention has multiple themed "dimension groups". A dimension group  is a group of multiple columns (for example, a location themed dimension group will have Country, State, City, ZIP). These definitions can be found in `pick_any_dimensions.py` they probably need to be revised as we move along in development for this feature.

## PR Checklist

### Front-End


### Separation of Concerns


### Tests


### Migrations
N/A
